File: IHIS/IHIS.Framework/IHIS.CloudConnector/Contracts/Models/Nuro/NuroOUT1001U01GetDepartmentInfo.cs
using System;

namespace IHIS.CloudConnector.Contracts.Models.Nuro
{
    [Serializable]
    public class NuroOUT1001U01GetDepartmentInfo
    {
        private String _gwa;
        private String _gwaName;
        private String _buseoCode;

        public String Gwa
        {
            get { return this._gwa; }
            set { this._gwa = value; }
        }

        public String GwaName
        {
            get { return this._gwaName; }
            set { this._gwaName = value; }
        }

        public String BuseoCode
        {
            get { return this._buseoCode; }
            set { this._buseoCode = value; }
        }

        public NuroOUT1001U01GetDepartmentInfo() { }

        public NuroOUT1001U01GetDepartmentInfo(String gwa, String gwaName, String buseoCode)
        {
            this._gwa = gwa;
            this._gwaName = gwaName;
            this._buseoCode = buseoCode;
        }

    }
}

File: MED-CLOUD/med-data/src/main/java/nta/med/data/model/ihis/nuro/NuroOUT1001U01GetDepartmentInfo.java
package nta.med.data.model.ihis.nuro;

public class NuroOUT1001U01GetDepartmentInfo {
	private String gwa;
    private String gwaName;
    private String buseoCode;
	public NuroOUT1001U01GetDepartmentInfo(String gwa, String gwaName,
			String buseoCode) {
		super();
		this.gwa = gwa;
		this.gwaName = gwaName;
		this.buseoCode = buseoCode;
	}
	public String getGwa() {
		return gwa;
	}
	public void setGwa(String gwa) {
		this.gwa = gwa;
	}
	public String getGwaName() {
		return gwaName;
	}
	public void setGwaName(String gwaName) {
		this.gwaName = gwaName;
	}
	public String getBuseoCode() {
		return buseoCode;
	}
	public void setBuseoCode(String buseoCode) {
		this.buseoCode = buseoCode;
	}
    
}
